src/HotChocolate/Data/test/Data.Sorting.Tests/SortInputTypeTests.cs

```diff
@@ -147,6 +148,21 @@ public void SortInput_AddName()
         schema.MatchSnapshot();
     }
 
+    [Fact]
+    public void SortInput_AddField()
+    {
+        // arrange
+        // act
+        var schema = CreateSchema(
+            s => s.AddType(
+                new SortInputType(
+                    d => d.Field("x").Type<DefaultSortEnumType>()
+                        .ExtendWith(x => x.Definition.Handler = new MatchAnyQueryableFieldHandler()))));
```

Comment:
```suggestion
        var schema = CreateSchema(
            s => s.AddType(
                new SortInputType(
                    d => d.Field("x").Type<DefaultSortEnumType>()
                        .ExtendWith(
                            x => x.Definition.Handler = new MatchAnyQueryableFieldHandler()))));
```

Try to keep lines <= 100 characters.